sandbox-open-imis/charts/gateway/templates/service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: open-imis-gateway
```

Comment:
Why not use variables from values file here as well like for deployment?

Reply:
I am new to Helm Charts and it's not obvious for me when it is better to use variables.
I applied suggestion.
Thank you.